cfgov/jinja2/v1/_includes/organisms/mega-menu.html

```diff
@@ -45,13 +45,14 @@
 {% macro _nav_list( nav_depth, nav_groups ) %}
 {%- for group in nav_groups %}
 <div class="{{ _classes( nav_depth, '-list' ) }}">
-  {% if group.title %} 
-  <h3 class="h5 {{' hide-title' if group.hide_title else ''}}">
-      {{group.title | safe}}
-  </h3>
-  {% endif %}
-  <ul class="m-list
-             m-list__unstyled">
+    {% if group.title %} 
+    <h3 class="h5 {{' hide-title' if group.hide_title else ''}}"
+        {% if group.hide_title %} aria-hidden="true"{% endif %}>
```

Comment:
Perhaps reformat this new conditional to use the same style as on the line above it, for consistency? I.e.,

`{{ 'aria-hidden="true"' if group.hide_title else '' }}`